META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,21 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Functional Java
+Bundle-SymbolicName: org.functionaljava
+Bundle-Version: 4.3.0.qualifier
```

Comment:
@dobesv I don't know about Eclipse's use of manifests, but should the bundle version above be 4.2 for the last release?